lib/common/schemas.c

```diff
@@ -1180,6 +1180,19 @@ pcmk__update_schema(xmlNode **xml, const char *max_schema_name, bool transform,
     return rc;
 }
 
+// public wrapper
+int
+pcmk_update_configured_schema(xmlNode **xml)
+{
+    int rc = pcmk_rc_ok;
+//     if (){
+//         pcmk__update_configured_schema(xml, true);
+//     } else {
+//         pcmk__update_configured_schema(xml, false);
+//     }
```

Comment:
Just always pass true -- this is a typical library API and not a high-level one that returns an XML api-result, so it should just always log